tests/cugraph/test_basics.py

```diff
@@ -1,14 +1,9 @@
 # NOTE(vibwu): Currently cugraph must be imported before torch to avoid a resource cleanup issue.
 #    See https://github.com/rapidsai/cugraph/issues/2718
-import unittest
-
-import backend as F
 import cugraph
-import numpy as np
-import pytest
+import backend as F
```

Comment:
can you add a todo here to explain why we need to import cugraph first, and link to the ticket to resolve it?

Reply:
at the top of this file, such note has already been there. so we don't need to add more.